lib/presentation/widget/upload_request_creation/upload_request_creation_widget.dart

```diff
@@ -169,7 +169,7 @@ class _UploadRequestCreationWidgetState extends State<UploadRequestCreationWidge
                         ..addOnSizeTypeNotifier(_model.totalFileSizeTypeNotifier))
                       .build()
                     : null)
-                ..addPasswordProtectedInput(creationState.uploadRequestCreation?.passwordProtected != null
+                ..addPasswordProtectedInput(creationState.uploadRequestCreation?.protectPasswordSetting != null
```

Comment:
Please careful on it, maybe conflict with the older version of back-end

Reply:
i tested with old versions and it works as intented